Demo/src/main/java/com/paypal/android/ui/card/DateString.kt

```diff
@@ -58,6 +60,27 @@ data class DateString(private val rawDate: String) {
         this.didPadZero = padZero
         this.didAddSlash = addSlash
         this.formatted = formatted
+
+        // TODO: handle invalid date string
+        var formattedMonth = ""
+        var formattedYear = ""
+
+        val dateStringComponents = formatted.split("/")
+        if (dateStringComponents.isNotEmpty()) {
+            formattedMonth = dateStringComponents[0]
```

Comment:
Take it or leave it: do we care about handling months similar to year where if you enter a single digit month like `1` for January padding it with a leading `0`? Not sure if the API handles this or if we'd want to.

Reply:
Yeah we do pad it in this class it's a couple lines up. It's just weird because of the way Compose works we're using a class called a `VisualTransformation` to turn `"125"` into `"01/25"` when we're rendering the text, but the actual value of the text field in this case will be `"125"`.

Reply:
Here's a link to [the padding part](https://github.com/paypal/paypal-android/blob/736e42b473c41379e3c6ea9aeed2412f7ae34132/Demo/src/main/java/com/paypal/android/ui/card/DateString.kt#L47).

Reply:
Oh dope! Didn't see it trapped in that non-expanded part.